src/text-editor.coffee

```diff
@@ -609,15 +609,19 @@ class TextEditor extends Model
   # * "<filename>" when it is the only editing buffer with this file name.
   # * "<filename> — <unique-dir-prefix>" when other buffers have this file name.
   #
+  # * `textEditors` An optional {Array} of the other {TextEditor} instances that
```

Comment:
I feel like we should default to using the global but deprecate it. Give package authors at least a couple minor releases to catch up.